### PR TITLE
Fix filter menus

### PR DESCRIPTION
**Pull request by erikjv**
- replaced "No filter" option text with "All", to avoid the "No filter
  is not enabled" situation
- replace the "Filter" label on the button with "1 Filter"/"2 Filters"
  when a filter is active, so a user can immediately see that without
  having to open the filter pop-up

### CODE REVIEW

File: src/gui/issueswidget.cpp [hunk 1]
```diff
@@ -281,6 +295,16 @@ void IssuesWidget::slotItemCompleted(const QString &folder, const SyncFileItemPt
     _model->addProtocolItem(ProtocolItem { folder, item });
 }
 
+void IssuesWidget::filterDidChange()
+{
+    int filterCount = _statusSortModel->filterCount();
```
Comment (TheOneRing) [inline src/gui/issueswidget.cpp:300]: I can't really follow what why count here and why we increase it if the regexp is empty.
Could you add a comment?

File: changelog/unreleased/9513 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Bugfix: improve filter pop-up menu and button
+    
+- replaced "No filter" option text with "All", to avoid the "No filter is not enabled" situation
+- replace the "Filter" label on the button with "1 Filter"/"2 Filters" when a filter is active, so a user can immediately see that without having to open the filter pop-up
+
```
Comment (TheOneRing) [inline changelog/unreleased/9513:5]: Add a url to this pr

File: changelog/unreleased/9513 [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+https://github.com/owncloud/client/issues/9425
```
Comment (TheOneRing) [inline changelog/unreleased/9513:6]: Maybe merge the entry with 9425?

**Review by TheOneRing — CHANGES_REQUESTED** [review]
As discussed

**Comment by erikjv** [comment]
sigh. something weird happened with this MR. Should be fine now.

File: src/gui/protocolwidget.cpp [hunk 4]
```diff
@@ -94,18 +95,28 @@ ProtocolWidget::~ProtocolWidget()
...
     QTimer::singleShot(0, menu, [menu] {
         menu->popup(QCursor::pos());
     });
     return menu;
 }
 
+QString ProtocolWidget::filterButtonText(int filterCount)
+{
+    if (filterCount == 0) {
+        return tr("Filter");
```
Comment (TheOneRing) [inline src/gui/protocolwidget.cpp:112]: Hmm https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals

**Pull request merged**